html/browsers/windows/auxiliary-browsing-contexts/resources/close-opener.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<meta charset="utf-8">
+<html>
+<body>
+<p>This window should close its opener.</p>
+<script>
+function closeOpener () {
+  window.addEventListener('storage', e => {
+    if (e.key === 'closeAll') {
+      window.close();
+    }
+  });
+  if (window.opener) {
+    window.opener.close();
+
+    // Give the browsing context a chance to dispose of itself
+    function waitForContextDiscard () {
+      if (window.opener === null) {
+        return localStorage.setItem('openerIsNull', 'true');
+      }
+      return setTimeout(waitForContextDiscard);
```

Comment:
Hmm maybe it's OK. I mostly expected a longer timeout than 0ms and indeed the browser will quickly throttle to 4ms.

FYI: The use of `setTimeout` is generally not OK in tests, `t.step_timeout` should be used instead; but this is not a test file so it I suppose it is OK here.